src/coreclr/pal/inc/pal.h

```diff
@@ -2855,7 +2855,7 @@ LocalAlloc(
        IN SIZE_T uBytes);
 
 PALIMPORT
-HLOCAL
+VOID
```

Comment:
This will introduce a difference from the Windows signature of the API. We can as well delete LocalAlloc / LocalFree from the PAL and ifdef out the few places where these are still used to use malloc/free on non-Windows.

Reply:
Deleted the local.cpp. #63850 will supersede `bsweep` changes.